vagrant-post-script/install_other_utils.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "[Vagrant] ###################     Installing nvm"
+curl -sS -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
```

Comment:
https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh is the newest version